website_event_baidu_map/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Website Event Baidu Map',
+    'version': '8.0.1.0.1',
```

Comment:
You can probably test it in v9 and it might be compatible. If this is the case you can create a second PR for v9.

Same goes for the other Baidu Map module